Sources/FactoryKit/FactoryKit/Aliases.swift

```diff
@@ -0,0 +1 @@
+../../Factory/Factory/Aliases.swift
```

Comment:
@hmlongco after running `generate-symlinks` this was added, not sure if it's needed or not